src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -239,7 +239,7 @@ public static void Title_Get_Windows_NoNulls()
         if (PlatformDetection.IsWindowsNanoServer)
         {
             // Nano server titles are currently broken
-            Assert.NotEqual(trimmedTitle, title);
+            Assert.Contains(title, trimmedTitle);
```

Comment:
The comment says that the titles are broken.  But the title now contains the one we expect?  Are they actually equal now, and if so, can we get rid of this special-case entirely?

Reply:
Please refer to test failure: https://github.com/dotnet/corefx/issues/34717

```
Assert.NotEqual() Failure\r
Expected: Not \"\\0峴\"\r
Actual:   \"\\0峴\"
```

I merged the two cases into the one Contains assertion and removed comment to avoid confusion

Reply:
> can we get rid of this special-case entirely?

I'm fine with that too

Reply:
Without knowing what the original issue was (it's not clear) I'm not sure whether `Contains` is sufficient to test it. E.g, if the bug was originally that the string ended with `\0\0` then `Contains` would not tell us if that bug returned.